### PR TITLE
Vo1ha solutions

### PR DESCRIPTION
**Pull request by vo1ha**
Hello! Day_1 and Day_2 solutions are added.

### CODE REVIEW

**Comment by vo1ha** [comment]
Hello @zaebee !
Could you please review solutions?

File: solutions/Day_2/Day_02.ipynb [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+   "source": [
+    "class StringOperations:\n",
+    "    def __init__(self):\n",
+    "      self.default_string = 'NULL'\n",
+    "\n",
+    "    def getString(self):\n",
+    "      user_str = input('Enter a string: ')\n",
+    "      return user_str\n",
+    "\n",
+    "    def printString(self, input_str):\n",
```
Comment (zaebee) [inline solutions/Day_2/Day_02.ipynb:125]: `printString` method should not take argument. Just return value assigned in the `__init__` method, kile:
```python
def getString(self):
  self.default_string = input()

def printString(self):
  return self.default_string

File: solutions/Day_3/Day_03.ipynb [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+    }
+   ],
+   "source": [
+    "# Get input from the console and split by commas\n",
+    "str_input = input().split(',')\n",
+    "# Convert each element in a list to int with base = 2 to interpret binary numbers\n",
+    "int_input = [int(i,2) for i in str_input]\n",
+    "\n",
+    "div_by_5 = []\n",
+    "for i in int_input:\n",
```
Comment (zaebee) [inline solutions/Day_3/Day_03.ipynb:121]: As option, you can use List Comprehension
```python
div_by_5 = [format(i, 'b') for i in int_input if i % 5 == 0]
```

File: solutions/Day_3/Day_03.ipynb [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+    "# Get input from the console and lower() possible letters to have them in the same format\n",
+    "word = input().lower()\n",
+    "\n",
+    "# Variables to calculate future letters and digits\n",
+    "letter_count = 0\n",
+    "digit_count = 0\n",
+    "\n",
+    "# Loop through each element in an input string and compare with letters \n",
+    "# and with numbers passed as strings - it is possible because of ASCII\n",
+    "for i in word:\n",
```
Comment (zaebee) [inline solutions/Day_3/Day_03.ipynb:285]: Another way is yo use `isalpha` and `isnumenric` method of str, like:
```python
for i in word:
  if i.isalpha():  # returns True if alphabet
    letter_count += 1
  elif i.isnumeric():  # returns True if numeric
    digit_count += 1
```

**Review by zaebee — APPROVED** [review]
Great job!

**Pull request merged**